src/fe-common/core/fe-messages.c

```diff
@@ -538,6 +538,20 @@ static void sig_message_invite(SERVER_REC *server, const char *channel,
 	g_free(str);
 }
 
+static void sig_message_invite_other(SERVER_REC *server, const char *channel,
+				     const char *invited, const char *nick, const char *address)
+{
+	char *str;
+	int level = MSGLEVEL_INVITES;
+
+	ignore_check_plus(server, nick, address, channel, invited, &level, FALSE);
+
+	str = show_lowascii(channel);
```

Comment:
TIL about this function! Seems like the point is to prevent people from introducing color codes in channel names to trick others into joining channels they don't actually trust, which makes sense for incoming invites for you.

For invites to other people in channels you already are part of, it doesn't make as much sense.

But harmless, so I don't mind if you keep it this way.